aws/internal/envvar/funcs_test.go

```diff
@@ -48,3 +48,119 @@ func TestGetWithDefault(t *testing.T) {
 		}
 	})
 }
+
+func TestRequireOneOf(t *testing.T) {
+	envVar1 := "TESTENVVAR_FAILIFALLEMPTY1"
+	envVar2 := "TESTENVVAR_FAILIFALLEMPTY2"
```

Comment:
Nit: Just to prevent parallel testing issues (aligning naming)

```suggestion
	envVar1 := "TESTENVVAR_REQUIREONEOF1"
	envVar2 := "TESTENVVAR_REQUIREONEOF2"
```